README.md

```diff
@@ -0,0 +1,79 @@
+# SystemLink Client API Documentation
+
+This repository contains getting started guides, examples, and reference
+documentation for the SystemLink Client NuGet packages. These packages support
+.NET Standard 2.0, meaning they support both .NET Core 2.0 and .NET Framework
+4.6.1 with Visual Studio 2017 or later.
+
+## Client APIs
+
+Each client API has its own NuGet package for interacting with a specific
+SystemLink service or collection of related services:
+
+| Service | Package name                                   | Package
+|---------|------------------------------------------------|------------------
+| Tag     | NationalInstruments.SystemLink.Clients.Tag     | Not yet published
+| Message | NationalInstruments.SystemLink.Clients.Message | Not yet published
+
+## Getting Started
+
+To use a SystemLink client in your application, add a reference to the NuGet
+package for the service or services you want to interact with.
+
+#### .NET Core
+
+```
+dotnet add package NationalInstruments.SystemLink.Clients.Tag
+```
+
+#### .NET Framework
+
+Use the NuGet Package Manager to add a reference to the package, or the NuGet
```

Comment:
Question: Would the user be able to add a reference to the package using the NuGet console too? If so, I suggest restructuring this sentence a little.
"Use the NuGet Package Manager or the NuGet console to add a reference to the package."

Reply:
Yes. I updated the sentence with your suggestion, which makes a lot more sense.